index.js

```diff
@@ -67,9 +64,13 @@ var Dropzone = React.createClass({
       height: this.props.size || 100,
       borderStyle: this.state.isDragActive ? "solid" : "dashed"
     };
- 
+
+    if (this.props.className) {
+      style = this.props.style;
+    }
+
```

Comment:
updated so that style is always props.style OR the hardcoded default, however, if a className is explicitly passed in, the styling *has* to come either from the passed in style, or from plain CSS/LESS/SASS/Stylus/whatever.